rclpy/rclpy/__init__.py

```diff
@@ -52,14 +52,20 @@
 from rclpy.utilities import ok  # noqa: F401 forwarding to this module
 from rclpy.utilities import shutdown as _shutdown
 from rclpy.utilities import try_shutdown  # noqa: F401
+from rclpy.utilities import get_default_domain_id
 
 # Avoid loading extensions on module import
 if TYPE_CHECKING:
     from rclpy.executors import Executor  # noqa: F401
     from rclpy.node import Node  # noqa: F401
 
 
-def init(*, args: Optional[List[str]] = None, context: Context = None) -> None:
+def init(
```

Comment:
Can we add test with specific domain_id in https://github.com/ros2/rclpy/blob/master/rclpy/test/test_init_shutdown.py?

Reply:
OK. I will supplement test codes.